### DISCUSSION TITLE
How to compile the firmware to use with an inductive probe?

### DISCUSSION THREAD

**Discussion by Solarflame5** [mriscoc/Ender3V2S1#1339] [Q&A] [answered]
I got a cheap inductive probe that outputs 5v on its signal pin when it detects the bed, I'm planning to install it on my printer but I'm really confused on how to set up the firmware, none of the available precompiled files or the configurations in the [Special_Configurations](https://github.com/mriscoc/Special_Configurations) repository seem to be designed for a simple probe like this,

**Comment by mriscoc** [comment]
You need to contact the manufacturer of that probe and request the settings for configure Marlin, then those settings can be ported to this firmware.

**Reply by Solarflame5** [reply, marked as answer]
Thanks for the information, I modified the EZABL configuration with some trial and error and got it working perfectly. Here is the config file in case anyone needs it in the future: https://github.com/Solarflame5/Ender3V2-InductiveProbe/blob/Ender3V2S1-Released/configurations/Ender3V2-422-IND/Configuration.h

**Reply by mriscoc** [reply]
Could be useful for other users if you post the details about that inductive probe and summarize what changes were necessary to the EZABL configuration. Maybe a new Json can be created to that probe.

**Reply by Solarflame5** [reply]
That's a good idea, I'll do a small write-up about it later.

**Reply by Solarflame5** [reply]
I wrote a short tutorial available here https://github.com/Solarflame5/Ender3V2-Inductive-Probe-Tutorial
